### ISSUE TITLE
Improve monorepo support with Babel

### ISSUE THREAD

**Issue by kripod** [wessberg/rollup-plugin-ts#95]
Previously, when using `rollup-plugin-babel`, I could specify [`{ 'rootMode': 'upward' }`](https://babeljs.io/docs/en/options#rootmode) as a configuration option, as shown below:

![image](https://user-images.githubusercontent.com/14854048/80405000-4a4fd400-88c2-11ea-813f-484700765cf4.png)

Unfortunately, `rollup-plugin-ts` doesn't seem to honor that option at this time, so I have to specify the location of the `babel.config.js` file in my repository root manually, with `babelConfig: '../..'`.

**Comment by wessberg** [comment]
Hey there. This should work:

```typescript
ts({
  transpiler: "babel",
  babelConfig: {
    rootMode: "upward"
  }
})
```

The `babelConfig` option takes whatever you would normally be able to pass to babel as a `TransformOption`.

I'm closing this issue since, but feel free to comment again in case you have any additional questions, or if it doesn't work as you expect it to.

**Comment by kripod** [comment]
I've tried that, but it doesn't work, unfortunately. Even the [related tests](https://github.com/wessberg/rollup-plugin-ts/blob/444846bc4d2ef7784bf16e829d59336b288c62e6/test/babel.test.ts#L46-L114) are written by setting the `cwd`, leading to a much less convenient process:

![image](https://user-images.githubusercontent.com/14854048/81936556-c81a1c00-95f2-11ea-9787-17ef60555099.png)

**Comment by wessberg** [comment]
If you can provide me with a minimal repro, that would be a tremendous help, because then I can set up a test case and get it fixed.

**Comment by kripod** [comment]
I’m going to use this approach in a project which isn’t released yet. I’ll get back to you in a few days, as soon as the code it published.

**Comment by kripod** [comment]
The project I’ve mentioned before [is now released](https://twitter.com/kripod97/status/1262101505383059456), along with a [rollup config to test with](https://github.com/kripod/otion/blob/master/packages/otion/rollup.config.js).

**Comment by wessberg** [comment]
Hey there!

I don't know why it took me so long to address this issue, but it has been fixed as part of [v1.2.30](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.2.30). I recommend reading the release notes if you would like to have insight into why it didn't work before.
